website/server/controllers/api-v3/tasks.js

```diff
@@ -629,7 +642,7 @@ api.addTagToTask = {
     if (validationErrors) throw validationErrors;
 
     let taskId = req.params.taskId;
-    let task = await Tasks.Task.findByIdOrAlias(taskId, user._id, { userId: user._id });
```

Comment:
removing the third argument this route can be used by anyone that knows the taskId, even if they don't have the right because the query will be `{_id: taskId}` when taskId is an UUID